x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/cohere/CohereServiceSettings.java

```diff
@@ -110,23 +110,25 @@ public CohereServiceSettings(StreamInput in) throws IOException {
         modelId = in.readOptionalString();
     }
 
-    public URI getUri() {
+    public URI uri() {
         return uri;
     }
 
-    public SimilarityMeasure getSimilarity() {
+    @Override
+    public SimilarityMeasure similarity() {
```

Comment:
I'm switching these to be inline with the other services. We had a bug where we weren't overriding the default implementation in the `ServiceSettings` interface which defaults to returning `null`.